nsis/SectionVersionCheck.nsh

```diff
@@ -7,17 +7,17 @@ Function UninstallPrevious
 ; $needle: String to search for
 ; $Haystack: String to look in
 
-; This will become the primary version check for 1.6.0 and later
+; This will become the primary version check
 ReadRegStr $R1 HKLM "${INSTDIR_REG_KEY}" "DisplayVersion"
 ReadRegStr $R2 HKLM Software\PCSX2 "Install_Dir"
 ${If} $R1 != ""
 ${AndIf} $R2 != ""
     Goto UserPrompt
 ${EndIf}
 
-; Search for 1.4.0
-ReadRegStr $R3 HKLM "${INSTDIR_REG_KEY}" "Uninst-pcsx2 1.4.0Directory"
-${StrContains} "$2" "1.4.0" "$R3"
+; Search for previous/older version
+ReadRegStr $R3 HKLM "${INSTDIR_REG_KEY}" "Uninst-pcsx2 1.6.0Directory"
```

Comment:
This block of code is actually no longer necessary and won't work with any version past 1.4 - it can be removed completely unless we want to detect 1.4 installations.

 The uninstall path no longer contains a version number as it reduces the chance of possible bugs, ect. The "primary verison check" is compatible with all future PCSX2 versions going forward as it detects the presence of the new version string and an install path.